src/ui/public/doc_viewer/viewers/json.js

```diff
@@ -0,0 +1,27 @@
+import _ from 'lodash';
+import angular from 'angular';
+import 'ace';
+import docTableDetailViewsRegistry from 'ui/registry/doc_table_detail_views';
+
+import jsonHtml from './json.html';
+
+docTableDetailViewsRegistry.register(function () {
+  return {
+    title: 'JSON',
```

Comment:
I would prefer if these objects didn't mix together the properties needed for the `<render-directive>` definition and extra properties needed for the `docViewer`. Maybe the object should look more like this?

```js
return {
  title: ..
  order: ...
  directive: {
    template: ...,
    controller() { }
  }
};
```